public/scripts/openai.js

```diff
@@ -1444,28 +1444,29 @@ export async function prepareOpenAIMessages({
  * Handles errors during streaming requests.
  * @param {Response} response
  * @param {string} decoded - response text or decoded stream data
+ * @param {boolean?} [supressToastr=false]
  */
-function tryParseStreamingError(response, decoded) {
+export function tryParseStreamingError(response, decoded, supressToastr = false) {
```

Comment:
We usually name this `quiet`. Just for consistency.
And as an optional argument - and knowing how easily exported functions grow -, I'd prefer it to be passed in with the optional args syntax.
```suggestion
export function tryParseStreamingError(response, decoded, { quiet = false } = {}) {
```

Reply:
Fixed